arch/arm64/boot/dts/qcom/msm8996.dtsi

```diff
@@ -1905,9 +1909,10 @@
 		vdd-supply = <&pm8994_s2_corner>;
 		vdda18-supply = <&pm8994_l12>;
 		vdda33-supply = <&pm8994_l24>;
-		qcom,vdd-voltage-level = <1 5 7>;
+		qcom,vdd-voltage-level = <1 4 7>;
 		qcom,tune2-efuse-bit-pos = <21>;
 		qcom,tune2-efuse-num-bits = <4>;
+		qcom,enable-dpdm-pulsing;
```

Comment:
4596bb98d18ea74008ab5ea796c423f26b18960b

Reply:
Dropped since nothing uses it (only 3.18 `drivers/usb/phy/phy-msm-qusb.c` used it)